### PR TITLE
feature: query FTX balances

### PR DESCRIPTION
**Pull request by ycdesu**
ticket: #109

Hello @c9s , I have implemented the `QueryAccountBalances` of FTX exchange, and also extracted some helpers. I have a question about currency conversion (that's why I add a TODO in the comment). I wonder know is there any global currencies string list? I don't know how to convert the FTX currency into the global currencies. If there's a list, I can define a string map in the `convert.go` and do the conversion. Thanks for your reply.

### CODE REVIEW

**Comment by c9s** [comment]
we don't define global currency list, just pass the currency name in uppercase, so we don't have to maintain a list.

if they have special formatting, like XBTC something, we can just trim the prefix.

**Comment by ycdesu** [comment]
Ok. Ftx seems return currency in uppercase. I will merge first and convert the currency if necessary in the future.

File: pkg/exchange/ftx/exchange.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+const (
+	restEndpoint       = "https://ftx.com"
+	defaultHTTPTimeout = 15 * time.Second
+)
+
+type Exchange struct {
+	rest *restRequest
+}
+
+func NewExchange(key, secret string, subAccount string) *Exchange {
```
Comment (c9s) [inline pkg/exchange/ftx/exchange.go:23]: is subaccount optional?

Comment (ycdesu) [reply pkg/exchange/ftx/exchange.go:23]: Yes. If your credential is created for all accounts, you don't have to specify that. If it's only for subaccount, you just specify it.

**Review by c9s — APPROVED** [review]
looks good

**Pull request merged**